### ISSUE TITLE
inconsistent file description 

### ISSUE THREAD

**Issue by Jaywantstocode** [dilevin/computer-graphics-raster-images#53]
hi,
in demosaic.h it says the colors are BGGR
```
// Given a mosaiced image (interleaved BGGR colors in a single channel), created
// a 3-channel rgb image.
```
however, everywhere else its described as GBRG. I was wondering which one is correct, or do they just mean the same pattern as
G B
R G

**Comment by Jaywantstocode** [comment]
#39 
I guess based on this issue it can be both. It would still be good if i can get a clarfication!

**Comment by dilevin** [comment]
yes correct we won't deduct marks as long as code is consistent (so pick either BGGR or GBRG and stick with it)
